README.md

```diff
@@ -1 +1,131 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Bridge-the-gap] <a name="about-project"></a>
+
+**[Bridge-the-gap]** Is a simple project that compounds of a webpage (home and about page) for a mentorship is set to take place in Kenya and basically it contains information about the mentorship ie speakers,program and about the organisation behind the whole meeting.Here is a brief video of how it looks on both the desktop and mobile. https://www.loom.com/share/07886dee04764432b037de466af1e97c
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML.
+- CSS.
+- Javascript
+### Key Features <a name="key-features"></a>
+
+- **[mobile version]**
+- **[Desktop version]**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://mercymugambi.github.io/capstone_project_mentorshipProgram/)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+- vscode.
+- git.
+- Knowledge on HTML and CSS and javascript.
+
+### Setup
+Clone this repository to your desired folder:
+
+Navigate to the  directory where you  want to clone this project with your terminal.
+Clone this repository using the following commands:
+`https://github.com/mercymugambi/capstone_project_mentorshipProgram.git`
+
+
+### Install
+Install this project with this command in your terminal:
+`cd capstone_project_mentorshipProgram`
+
+
+### Usage
+
+To run the project, execute the following command in your terminal:
+`code .`
+
+### Run tests
+
+To run tests,
+
+Run it in your local server.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+## Mercy Mugambi
+
+- Github: [@mercymugambi](https://github.com/mercymugambi)
+- Twitter: [@mercymugambi](https://twitter.com/mercymugambi15)
+- Linkedin: [mercy mugambi](https://www.linkedin.com/in/mercymugambi)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Creating and linking consequent pages namely:
+   - Sponsor
+   - Join
+   - News
+   - campaign
+- Adding animations to the site so that the site is more engaging.
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+- Contributions, issues, and feature requests are welcome!
+
+- Feel free to check send me a message using the social media accounts stated above .
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+- If you like this project, give it a star .
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like appreciate Cindy Shin in Behance for the brilliant design that maps the base of this project.
```

Comment:
[OPTIONAL] Well done on the `README` template but you could also add a link to Cindy Shin's profile and Work on Behance, this could be like below:
```
I would like to give ppropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```